### PR TITLE
fix duplicate method in G::A::A::QueryComplexity

### PR DESCRIPTION
**Pull request by gfx**
AST::QueryComplexity has two `#result` methods so this PR fixes it.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ Layout/EndAlignment:
 Lint/UselessAssignment:
   Enabled: true
 
+Lint/DuplicateMethods:
+  Enabled: true
```
Comment (gfx) [inline .rubocop.yml]: This should be helpful anyway.

File: lib/graphql/analysis/ast/query_complexity.rb [hunk 2]
```diff
@@ -57,7 +57,8 @@ def on_leave_fragment_spread(node, _, visitor)
           visitor.leave_fragment_spread_inline(node)
         end
 
-        def result
+        # @return [Integer]
+        def max_possible_complexity
```
Comment (gfx) [inline lib/graphql/analysis/ast/query_complexity.rb]: This is the second, duplicate `#result`. I believe the renaming is needed, anyway.

File: .rubocop.yml (hunk 1 above)
Comment (rmosolgo) [reply .rubocop.yml]: 👏 !

**Comment by rmosolgo** [comment]
Thanks for improving this!

**Comment by gfx** [comment]
✌️

**Pull request merged**